src/cmr/concepts/__tests__/variable.test.js

```diff
@@ -0,0 +1,114 @@
+import Variable from '../variable'
+
+describe('Variable concept', () => {
+  beforeEach(() => {
+    jest.resetAllMocks()
+
+    jest.restoreAllMocks()
+  })
+
+  describe('Variable concept', () => {
+    describe('retrieve the parent collection', () => {
+      describe('if the nested fields `instanceInformation` or `scienceKeywords` are queried', () => {
+        test('Ensure that their child fields return in camelCase', () => {
+          const variable = new Variable({}, {}, {})
+          const variableConceptId = 'V1200000000-EDSC'
+          const jsonKeys = ['instanceInformation', 'scienceKeywords']
+          const jsonResponse = {
+            data:
+            {
+              hits: 1,
+              took: 9,
+              items: [
+                {
+                  long_name: 'sea surface subskin temperature',
```

Comment:
I'd prefer to just remove everything here that isn't getting parsed but, I wasn't sure if people thought it helped to have something that looks like a "real" response